### DISCUSSION TITLE
Adding a paragraph/footer note about Starfleet

### DISCUSSION THREAD

**Discussion by WengerK** [webmardi/webmardi.ch#72] [Ideas]
What about having a paragraph that encourage people to install/have Starfleet in order to never miss any Webmardi announce ?

https://starfleet.jolicode.com/

Waht do you think @Yago ?

**Comment by Yago** [comment]
Yes we can, but there is not a lot of going on in this platform 🤔 Also, there is already call to actions for Meetup and the newsletter. Too many notification solutions may lose you audience...

Do you have a paragraph in mind to promote Starfleet on the site?

**Reply by WengerK** [reply]
You're right ! Let's put this pending for now. We will discuss it with the comm' team :) thanks for your feedback
